README.md

```diff
@@ -24,10 +39,17 @@ composer require rapidez/blade-components
 And make sure these colors are present in your Tailwind config:
 ```js
 colors: {
-    neutral: '#334155',
-    inactive: '#64748b',
-    border: '#e7ebef',
-    disabled: '#ebe8de',
+    colors: {
+        primary: {
+            DEFAULT: '#2FBC85',
+            text: '#FFFFFF',
```

Comment:
shouldn't we need a secondary variable? and a secondary button aswell? If a project doesn't have a secondary which i highly doubt then you can just set your secondary color the same as your primary